<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainCliente</class>
 <widget class="QWidget" name="MainCliente">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>597</width>
    <height>438</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Main Cliente</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>10</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Scegli l'operazione:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_inserisci">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>60</y>
     <width>251</width>
     <height>101</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Inserisci Cliente</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_ricerca">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>180</y>
     <width>251</width>
     <height>101</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Ricerca Cliente</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_tessere">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>300</y>
     <width>251</width>
     <height>101</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Tessere</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
